tests/test_types.py

```diff
@@ -630,3 +631,30 @@ def test_deepcopy_of_singleton_fixed_type() -> None:
 
     for lhs, rhs in zip(list_of_fixed_types, copied_list):
         assert id(lhs) == id(rhs)
+
+
+@pytest.mark.parametrize(
+    "val, expected",
+    [
+        ("y", True),
+        ("yes", True),
+        ("t", True),
+        ("true", True),
+        ("on", True),
+        ("1", True),
+        ("n", False),
+        ("no", False),
+        ("f", False),
+        ("false", False),
+        ("off", False),
+        ("0", False),
+    ],
+)
+def test_strtobool_valid(val: str, expected: bool) -> None:
```

Comment:
nit: instead of parameterizing this test, we can run it in 1 test in a for-loop. Running too many parameterized tests will slow down the overall test runtime.

Reply:
Great suggestion @kevinjqliu, here you go!